gnome-initial-setup/demo/eos-demo-video.c

```diff
@@ -106,8 +118,13 @@ create_demo_window (GisDemoVideo *demo_video)
                            demo_video,
                            G_CONNECT_AFTER);
 
+  g_signal_connect_object (GTK_WIDGET (priv->main_window),
```

Comment:
Is "GTK_WIDGET ()" needed here?  If so, shouldn't it also be used on the previous call to g_signal_connect_object?

Reply:
Not strictly speaking no, though it never hurts (considering that the "realize" signal is on the widget itself).

That said, the follow-up commit connects to a signal only defined on GtkWindow, so this isn't relevant anymore and we can drop the cast.